docs/Containerfile

```diff
@@ -0,0 +1,13 @@
+# The resulting image can be used to

```

Comment:
should this just be an image on `quay.io/podman` that can be pulled and then use volume mounts ?  I'm sure you have a reason to not do this so I'm merely asking...

Reply:
It could be an image on `quay.io/podman` but that would require to setup the CI for it.
I have avoided it for simplicity and because generating the docs locally is usually one-off (so it's okay to wait 30 seconds). But of course, this can be changed.

Reply:
your reasoning is sound.  if I'm using this a developer to do work locally, would a volume mount be more efficient in a use case where you are iterating on things?

Reply:
I have added a better explanation in the Containerfile comment. The image `CMD` is expecting the source code to be bind-mounted in the container folder `/podman`.

Read/Write on `/mnt/c/` (the host folder mounted in a WSL VM) is extremely slow. Users try to avoid I/O operations under that folder as much as possible. Regarding the docs generation, we still want to copy back the generated HTML, so a volume is necessary.  I want to try with an overlay volume to see if the throughput is better, but we need to [address a bug first](https://github.com/containers/podman/issues/25988).